stor/tests/cassettes_py2/TestCopyTree/test_empty_path.yaml

```diff
@@ -0,0 +1,139 @@
+interactions:
+- request:
+    body: !!python/unicode '{"name": "TestCopyTree.test_empty_path.4662e943"}'
+    headers:
+      Content-Type:
+      - application/json
+      DNAnexus-API:
+      - 1.0.0
+      User-Agent:
+      - dxpy/0.278.0 (Darwin-18.2.0-x86_64-i386-64bit)
+    method: POST
+    uri: https://api.dnanexus.com/project/new
+  response:
+    body:
+      string: !!python/unicode '{"id":"project-FXFPP6801YyXv3Q32v06xKyz"}'
+    headers:
+      connection:
+      - keep-alive
+      content-length:
+      - '41'
+      content-type:
+      - application/json; charset=utf-8
+      date:
+      - Sun, 24 Mar 2019 03:20:57 GMT
+      server:
+      - nginx
+      x-content-type-options:
+      - nosniff
+      x-powered-by:
+      - Express
+      x-request-id:
+      - 1553397657042-395806
+    status:
+      code: 200
+      message: OK
+- request:
+    body: !!python/unicode '{}'
+    headers:
+      Content-Type:
+      - application/json
+      DNAnexus-API:
+      - 1.0.0
+      User-Agent:
+      - dxpy/0.278.0 (Darwin-18.2.0-x86_64-i386-64bit)
+    method: POST
+    uri: https://api.dnanexus.com/project-FXFPP6801YyXv3Q32v06xKyz/describe
+  response:
+    body:
+      string: !!python/unicode '{"id":"project-FXFPP6801YyXv3Q32v06xKyz","name":"TestCopyTree.test_empty_path.4662e943","class":"project","created":1553397657000,"modified":1553397657000,"billTo":"org-counsyl","level":"ADMINISTER","dataUsage":0,"sponsoredDataUsage":0,"region":"aws:us-east-1","summary":"","description":"","protected":false,"restricted":false,"downloadRestricted":false,"containsPHI":false,"createdBy":{"user":"user-akumar_counsyl"},"version":0,"storageCost":0,"pendingTransfer":null,"tags":[],"defaultInstanceType":"mem2_hdd2_x2","archivalState":"live","archivalProgress":null,"totalSponsoredEgressBytes":0,"consumedSponsoredEgressBytes":0,"atSpendingLimit":false}'
+    headers:
+      connection:
+      - keep-alive
+      content-length:
+      - '650'
+      content-type:
+      - application/json; charset=utf-8
+      date:
+      - Sun, 24 Mar 2019 03:20:57 GMT
+      server:
+      - nginx
+      x-content-type-options:
+      - nosniff
+      x-powered-by:
+      - Express
+      x-request-id:
+      - 1553397657159-954225
+    status:
+      code: 200
+      message: OK
+- request:
+    body: !!python/unicode '{"name": "TestCopyTree.test_empty_path.4662e943", "limit":
+      2, "level": "VIEW"}'
+    headers:
+      Content-Type:
+      - application/json
+      DNAnexus-API:
+      - 1.0.0
+      User-Agent:
+      - dxpy/0.278.0 (Darwin-18.2.0-x86_64-i386-64bit)
+    method: POST
+    uri: https://api.dnanexus.com/system/findProjects
+  response:
+    body:
+      string: !!python/unicode '{"results":[{"id":"project-FXFPP6801YyXv3Q32v06xKyz","level":"ADMINISTER","permissionSources":["user-akumar_counsyl"],"public":false}],"next":null}'
```

Comment:
why does this only end up returning one project?  Also shouldn't this be returning project name too?

Reply:
This call is a result of 'dest.canonical_project` [here](https://github.com/counsyl/stor/blob/master/stor/dx.py#L614). We call `_clonetree` and `_movetree` in `TestCopyTree.test_empty_path`. If it was reversed to `self.canonical_project == dest.canonical_project`, we'd not be making that dx call.